torch/fx/experimental/symbolic_shapes.py

```diff
@@ -159,7 +166,7 @@ def wrap_float(self, num):
         return SymNode(sympy.Float(num), self.shape_env, float, constant=num)
 
     def clone(self):
-        return SymNode(self.expr, self.shape_env, self.pytype, constant=self.constant)
+        return self
```

Comment:
see also https://github.com/pytorch/pytorch/pull/88230 but that PR is pretty wedged, so if this change doesn't cause test failures seems OK to roll it up here

Reply:
Reverting this change. It's causing partitioner failure 

```
ERROR [0.175s]: test_default_partitioner_output_tensor_shape_tensor (__main__.TestPartitioning)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/unittest/mock.py", line 1369, in patched
    return func(*newargs, **newkeywargs)
  File "/var/lib/jenkins/workspace/test/functorch/test_aotdispatch.py", line 679, in test_default_partitioner_output_tensor_shape_tensor
    compiled_outs = aot_function(
  File "/opt/conda/lib/python3.10/site-packages/functorch/_src/aot_autograd.py", line 806, in returned_function
    compiled_fn = create_aot_dispatcher_function(
  File "/opt/conda/lib/python3.10/site-packages/functorch/_src/aot_autograd.py", line 657, in create_aot_dispatcher_function
    aot_dispatch_autograd(flat_fn, fake_flat_tensor_args, aot_config)
  File "/opt/conda/lib/python3.10/site-packages/functorch/_src/aot_autograd.py", line 501, in aot_dispatch_autograd
    fw_module, bw_module = aot_config.partition_fn(fx_g, joint_inputs)
  File "/opt/conda/lib/python3.10/site-packages/functorch/_src/partitioners.py", line 181, in default_partition
    return _extract_fwd_bwd_modules(joint_module, saved_values, saved_sym_nodes=saved_sym_nodes)
  File "/opt/conda/lib/python3.10/site-packages/functorch/_src/partitioners.py", line 122, in _extract_fwd_bwd_modules
    fwd_module = fx.GraphModule(joint_module, fwd_graph)
  File "/opt/conda/lib/python3.10/site-packages/torch/fx/graph_module.py", line 368, in __init__
    self.graph = graph
  File "/opt/conda/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1564, in __setattr__
    super().__setattr__(name, value)
  File "/opt/conda/lib/python3.10/site-packages/torch/fx/graph_module.py", line 409, in graph
    self.recompile()
  File "/opt/conda/lib/python3.10/site-packages/torch/fx/graph_module.py", line 646, in recompile
    cls.forward = _forward_from_src(self._code, python_code.globals)
  File "/opt/conda/lib/python3.10/site-packages/torch/fx/graph_module.py", line 79, in _forward_from_src
    _exec_with_source(src, globals_copy)
  File "/opt/conda/lib/python3.10/site-packages/torch/fx/graph_module.py", line 73, in _exec_with_source
    exec(compile(src, key, 'exec'), globals)
  File "<eval_with_key>.127", line 7
    return [Invalid Node, sym_size, sym_size_7, primals_3, Invalid Node, primals_4, primals_3, primals_2, primals_1, sym_size]
            ^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?


```